StripeFinancialConnections/StripeFinancialConnections/Source/Native/Consent/ConsentLogoView.swift

```diff
@@ -59,7 +59,11 @@ final class ConsentLogoView: UIView {
 
     func animateDots() {
 #if targetEnvironment(simulator)
-        if ProcessInfo.processInfo.environment["UITesting"] != nil {
+        guard ProcessInfo.processInfo.environment["UITesting"] == nil else {
+            return
+        }
+
+        guard ProcessInfo.processInfo.environment["XCTestSessionIdentifier"] == nil else {
```

Comment:
This is similar to the above condition, but catches all XCTests environments rather than just UI tests. Thinking about it now, maybe we no longer need the above condition with this here 🤔

Reply:
feel free to delete if you are 100% sure that UITesting is unnecessary

I am personally not familiar with `XCTestSessionIdentifier` vs. `UITesting`

Reply:
Just checked, when UI Testing, the `XCTestSessionIdentifier` environment variable isn't set, so we'll keep both of these checks here for now 👍